marvin/core/context.py

```diff
@@ -0,0 +1,40 @@
+from marvin.core.status import Status
+from marvin.exceptions import ContextSkippedException
+from marvin.report import Publisher
+
+
+class Context(object):
+
+    def __init__(self, parent_context=None):
+        self._ctx = parent_context
+        self._publisher = self.ctx.publisher if self.ctx else Publisher()
+        self._sub_context_results = {
+            Status.PASS: 0,
+            Status.FAIL: 0,
+            Status.SKIP: 0
+        }
+
```

Comment:
Sure. I can wrap that in a `@property` decorated method. Can you think of a better name than `sub_context_results`?